### DISCUSSION TITLE
cloudflare workers 部署后下载文件报错

### DISCUSSION THREAD

**Discussion by usufu** [AlistGo/alist#709] [Q&A]
cloudflare workers 部署后下载文件报错，错误error1101，有大佬知道怎么回事嘛？

**Comment by xhofe** [comment]
需要看具体的报错，cf worker后台可以监听日志。

**Reply by usufu** [reply]
看了下，报json错误，不知道下一步该如何调试。
"outcome": "exception",
  "scriptName": null,
  "exceptions": [
    {
      "name": "SyntaxError",
      "message": "Unexpected token e in JSON at position 0",
      "timestamp": 1646874516882
    }
  ],

**Reply by xhofe** [reply]
是不是host最后面加了/，又或者没有加http[s]://协议头

**Reply by usufu** [reply]
没有，都按要求弄的，只不过HOST是IP:PORT 这种形式。http://IP:PORT 这样

**Reply by xhofe** [reply]
那应该是cf workder访问不了你的这个地址

**Reply by shaojs321** [reply]
遇到同样问题，日志一样，host也是填http://ip:port

**Comment by usufu** [comment]
经过研究 worker不支持带端口的方式

shaojs ***@***.***>于2022年3月13日 周日18:53写道：

**Reply by FmgX** [reply]
欸，我的是这样的

  "outcome": "exception",
  "scriptName": null,
  "exceptions": [
    {
      "name": "SyntaxError",
      "message": "Unexpected end of JSON input",
      "timestamp": 1649985695790
    }
  ],
  "logs": [],
host最后没有加/，也有协议头
但是在进行访问下载时会提示找不到服务器

**Comment by Arc-2023** [comment]
把ip:port用域名代理，只填域名就行了
